src/tasks/purchase/taskpurchase.h

```diff
@@ -52,7 +52,12 @@ class TaskPurchase final : public Task {
  private:
   Op m_op;
 #ifdef MZ_IOS
-  QString m_iOSReceipt;
+  // m_iOSData is the originalTransactionId for iOS 15+, and the receipt for
+  // earlier versions of iOS
+  QString m_iOSData;
+  // `@available(iOS 15, *)` is not available in C++, so this value must be
+  // passed in and saved
+  bool isOlderOS;
```

Comment:
nit: our coding convention prefixes member variables with `m_`, and I would recommend following that trend (or abandoning it entirely if we feel it's silly). I don't have a strong preference, but I just like consistency :)

Reply:
Good call, thank you. Will be updated on the next push. (I also like consistency!)